### PR TITLE
added cards and avatar with badge

### PR DESCRIPTION
**Pull request by mmanjuprasad**

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -221,12 +221,34 @@
   .card-content:hover{
       background-color: #f8f8f8;
```
Comment (anasm11) [inline styles.css:222]: better to use variable names for colors.

File: styles.css [hunk 2]
```diff
@@ -221,12 +221,34 @@
...
+    /* padding: 1.4rem; */
+    font-weight: 500;
+    font-size: 1.2rem;
+    position: relative;
+    top: -4rem;
+    color: white;
+    /* bottom: 20px; */
+    /* left: 10px; */
+    /* z-index: -auto; */
+    /* overflow: hidden; */
```
Comment (anasm11) [inline styles.css:249]: comments can be removed.

File: styles.css [hunk 3]
```diff
@@ -318,13 +340,41 @@
...
+  color:#f8f8f8;
+  background-color: rgb(19, 18, 18);
+  border-radius: 50%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  position: absolute;
+}
+
+.cibd{
```
Comment (anasm11) [inline styles.css:365]: naming can be improved.

File: index.html [hunk 4]
```diff
@@ -207,6 +207,156 @@ <h4>Horizontal card</h4>
...
+                    style="width: 967px; height: 763px; border:0; transform: scale(1); overflow:hidden;"
```
Comment (anasm11) [inline index.html:338]: inline styling can be avoided.

**Review by anasm11 — APPROVED** [review]
have suggested changes. The code looks good.

File: styles.css [hunk 5]
```diff
@@ -221,12 +221,34 @@
...
+}
   
   .author-details{
       padding: 1.4rem;
       font-weight: 500;
       font-size: 1.2rem;
   }
+
+  .author-details-overlay{
+    /* padding: 1.4rem; */
```
Comment (pradipgharge) [inline styles.css:240]: remove commented code

File: styles.css [hunk 6]
```diff
@@ -221,12 +221,34 @@
...
+    font-weight: 500;
+    font-size: 1.2rem;
+    position: relative;
+    top: -4rem;
+    color: white;
+    /* bottom: 20px; */
+    /* left: 10px; */
+    /* z-index: -auto; */
+    /* overflow: hidden; */
+
```
Comment (pradipgharge) [inline styles.css:250]: same here

File: styles.css [hunk 7]
```diff
@@ -318,13 +340,41 @@
   .position-relative{
     position: relative;
     display: inline-block;
+    z-index: -1;
   }
```
Comment (pradipgharge) [inline styles.css:346]: remove extra spacing

**Review by pradipgharge — APPROVED** [review]
Code looks good, but you have to remove the commented code to make it  look clean

**Pull request merged**